src/main/java/eu/europeana/metis/sandbox/config/SandboxConfig.java

```diff
@@ -76,4 +88,32 @@ String initialQueue() {
   ZipFileReader zipFileReader() {
     return new ZipFileReader();
   }
+
+  @Bean
+  ClasspathResourceResolver lsResourceResolver() {
+    return new ClasspathResourceResolver();
+  }
+
+  @Bean
+  SchemaProvider schemaProvider() {
+    return new SchemaProvider(PredefinedSchemasGenerator.generate(schemaProperties()));
+  }
+
+  private Properties schemaProperties() {
+    YamlPropertiesFactoryBean propertiesFactoryBean = new YamlPropertiesFactoryBean();
+    propertiesFactoryBean.setResources(new ClassPathResource("application.yml"));
```

Comment:
Now instead of using a properties file I am using the application.yml, and I am able to get the `PredefinedSchemasGenerator.generate` the properties it needs